_lessons/python-scatter/01b-profiling-mpi-map.md

```diff
@@ -84,5 +83,5 @@ As an example, "CPU instructions" present the usage of different instruction set
 
 ## Exercises
 
- * profile the scatter code under the openmp directory using 4 OpenMP threads. Increase the problem size by passing options `-nx 256 -ny 256 -nc 256` to *scatter.py* to make the test run longer.
+ * profile the scatter code under the openmp directory using 16 MPI tasks. Increase the problem size by passing options `-nx 256 -ny 256 -nc 256` to *scatter.py* to make the test run longer.
```

Comment:
Since we profile MPI, we want to change into MPI directory.

Reply:
Fixed. Thanks.